styles.css

```diff
@@ -0,0 +1,522 @@
+  background-position: 80px 0;
+  background-size: 80% 100%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.contact__me__title {
+  font-family: Poppins, sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  text-align: center;
+  line-height: 52px;
+  font-style: normal;
+  margin-top: 103px;
+  margin-bottom: 0;
+  color: #fff;
+}
+
+.contact__me__text {
+  font-family: Poppins, sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  font-style: normal;
+  text-align: center;
+  line-height: 28px;
+  color: #efebef;
+  margin-top: 12px;
+  width: 250px;
+}
+
+.contact__me__form {
+  display: flex;
+  flex-direction: column;
+  margin: 0 10%;
+}
+
+.input {
+  color: #172b4d;
+  height: 48px;
+  border-radius: 10px;
+  outline: none;
+  border: 1px solid #cfd8dc;
+  margin-top: 20px;
+  padding-left: 16px;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.text__area {
+  height: 170px;
+  margin-top: 18px;
+  padding-left: 16px;
+  padding-top: 15px;
+  border-radius: 10px;
+  outline: none;
+  border: 1px solid #cfd8dc;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.btn2 {
+  margin-top: 22px;
+  margin-bottom: 45px;
+  float: left;
+  width: 140px;
+  height: 48px;
+  margin-left: 24px;
+}
+
+.btn2:hover {
+  background-color: #7b83dd;
+}
+
+@media (min-width: 992px) {
+  .flex__container {
+    display: flex;
+  }
+
+  .flex__child {
+    flex: 1;
+    margin: 0 20px;
+  }
+
+  .flex__child:first-child {
+    margin-right: 20px;
+  }
+
+  .nav-link {
+    display: inline;
+    margin-top: 100px;
+  }
+
+  .front-page-body {
+    background-image: url("images/Header-bg@2x.png");
+    background-position: 10px 60px;
+    background-size: 100% 90%;
+  }
+
+  .intro h1 {
+    width: 920px;
+  }
+
+  .intro .intro-paragraph {
+    width: 750px;
+  }
+
+  .card {
+    margin: 5% 4%;
+  }
+
+  .card__image {
+    width: 105%;
+    height: 100%;
+  }
+
+  .about__content {
+    margin: 5%;
+  }
+
+  .connect {
+    padding: 0;
+    margin: 0;
+  }
+
+  .languages {
+    display: inline-block;
+    background-color: #f7f7f9;
+    background-size: 30%;
+  }
+
+  .languages p {
+    margin-left: 0;
+    width: auto;
+  }
+
+  .contact__me {
+    background-image: url("images/contact-form-background-shapes.png");
+    background-size: 90% 100%;
+  }
+
+  .contact__me__text {
+    width: 568px;
+  }
+
+  .contact__me__form {
+    display: flex;
+    flex-direction: column;
+    margin: 0 10%;
+    width: 447px;
+  }
+
+  .input {
+    border-radius: 0;
+  }
+
+  .text__area {
+    border-radius: 0;
+  }
+
+  .btn2 {
+    margin: 22px auto;
+  }
+}
+
+@media only screen and (max-width: 1100px) {
+  .nav-menu {
+    position: fixed;
+    left: -100%;
+    top: 5rem;
+    flex-direction: column;
+    background-color: #fff;
+    width: 100%;
+    border-radius: 10px;
+    text-align: center;
+    transition: 0.3s;
+    box-shadow: 0 10px 27px rgba(0, 0, 0, 0.05);
+  }
```

Comment:
[OPTIONAL]
- I  would strongly recommend adding `display: none` to the `nav-menu` class in the mobile version. The `nav-menu` is still displayed on the mobile version.
- Please. Let's make the width of form inputs 100% in the mobile version to match the figma design.
<img width="157" alt="image" src="https://user-images.githubusercontent.com/77190203/150650370-1a4fed29-0483-4e5a-8aff-3fea5cd4f81f.png">